From 93ac885838a2c0cec567a0b5b7e1c37a8264dc5f Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Marc-Andr=C3=A9=20Lureau?= <marcandre.lureau@redhat.com>
Date: Thu, 16 May 2013 22:26:21 +0300
Subject: [PATCH] Silence unused arguments warnings

In file included from virt-viewer-session-spice.c:41:
gbinding.c: In function 'on_source_notify':
gbinding.c:381: warning: unused parameter 'gobject'
gbinding.c: In function 'on_target_notify':
gbinding.c:422: warning: unused parameter 'gobject'
gbinding.c: In function 'g_binding_init':
gbinding.c:709: warning: unused parameter 'binding'
---
 src/gbinding.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/gbinding.c b/src/gbinding.c
index 5b5757c..4b2f5b1 100644
--- a/src/gbinding.c
+++ b/src/gbinding.c
@@ -378,7 +378,7 @@ default_transform_from (GBinding     *binding,
 }
 
 static void
-on_source_notify (GObject    *gobject,
+on_source_notify (GObject    *gobject G_GNUC_UNUSED,
                   GParamSpec *pspec,
                   GBinding   *binding)
 {
@@ -419,7 +419,7 @@ on_source_notify (GObject    *gobject,
 }
 
 static void
-on_target_notify (GObject    *gobject,
+on_target_notify (GObject    *gobject G_GNUC_UNUSED,
                   GParamSpec *pspec,
                   GBinding   *binding)
 {
@@ -706,7 +706,7 @@ g_binding_class_init (GBindingClass *klass)
 }
 
 static void
-g_binding_init (GBinding *binding)
+g_binding_init (GBinding *binding G_GNUC_UNUSED)
 {
 }
 
